src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelTranscript.tsx

```diff
@@ -83,6 +80,33 @@ const TrackPanelTranscript = (props: Props) => {
     );
   };
 
+  const onChangeVisibility = () => {
+    const newStatus =
+      trackVisibilityStatus === Status.SELECTED
+        ? Status.UNSELECTED
+        : Status.SELECTED;
+
+    const updatedTranscriptTracks = {
+      ...transcriptTrackStatuses,
+      [transcript.stable_id]: newStatus
+    };
+    const visibleTranscriptIds = Object.keys(updatedTranscriptTracks).filter(
+      Boolean
+    );
```

Comment:
I am confused — why should `filter(Boolean)` work in this case? `Status.UNSELECTED` is a string.